index.html

```diff
@@ -0,0 +1,200 @@
+            <li class="Contact"><a href="#contact">Contact</a></li>
+            <i class='fas'>&#xf199;</i>
+        </ul>             
+    </nav>
+    </header>
+            <!-- <div class="top-section"> -->
+                <section class="top-section">
+                        <h1 class="title">Hi there.<br>I'm Peter</h1>
+                        <h3>I'm a software developer</h3>
+                        <p class="title-paragraph">I can help you build a project, feature or website Look through some of my work and experience! if you like whatyou see and have a project you need coded, don't hesitate to contact me. </p>
+                    <div class="socials">
+                        <ul class="social">
+                                <li class="twitter"><a href="https://www.twitter.com/jiyapete/"><i class="fa-brands fa-twitter" id="twitter"></i></a></li>
+                                <li class="link"><a href="https://www.linkedin.com/peteronyam/"><i class="fa-brands fa-linkedin" id="linkedin"></i></a></li>
+                                <li class="git"><a href="https://www.github.com/peteronyam"><i class="fa-brands fa-github" id="github"></i></a></li>
+                        </ul>
+                    </div>
+                </section>
+                </div>
+            <!-- </div> -->
+                <!-- work section start-->
+            <div class="recent-works">
+            <section class="services">
+                <div class="work-service">
+                <h2 class="services-tag">My Recent Work</h2>
+                <hr class="horizontal">
+                </div>
+                <img class="tag-image" src="./images/fbg.png" alt="placeholder">
+                <h3 class="service-header">Multi-Post Stories</h3>
+                <p class="services-paragraph">A daily selection of personalized read; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text</p>
+                <ul class="service-button">
+                    <li class="css-btn"><a id="css-btn" href="#">css</a></li>
+                    <li class="html-btn"><a id="html-btn" href="#">html</a></li>
+                    <li class="bootstrap-btn"><a id="bootstrap-btn" href="#">bootstrap</a></li>
+                    <li class="ruby-btn"><a id="ruby-btn" href="#">Ruby</a></li>
+                </ul>
+                <button type="button" class="See-project-btn">See Project</button>
+                <div class="pin">
+            <section class="printing" id="work1-section">
+                <h2 class="print-header" id="work1">Professional Art Printing Data</h2>
+                <p class="print-paragraph" id="work1-paragraph">
+                    A day selection of privately personalized reads; on accounts or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="print-button" id="work1-list">
```

Comment:
- [ ] As I mentioned above :point_up:  please stick to the Figma design but your work design is not the same as the Figma design there are some `margin` problems and also `My Recent Work` is not showing properly and also your cards are not same as Figma design so kindly fix these issues also. Check below  :point_down: the comparison between your design and Fidma Design :+1: 

**Your Design**
![screencapture-file-media-aamir-Workplace-Microverse-Code-Reviews-Projects-desktop-portfolio-index-html-2023-03-05-03_19_04 (1)](https://user-images.githubusercontent.com/45226848/222931384-18574ea6-1ce0-4d1b-9776-bab590ed6429.png)

**Figma Design**
![Screenshot from 2023-03-05 03-23-08](https://user-images.githubusercontent.com/45226848/222931402-3c1cb69b-7076-4a76-9e9a-21f95389c96a.png)